test/queries.js

```diff
@@ -0,0 +1,248 @@
+/* eslint-env mocha */
+
+'use strict'
+
+const assert = require('assert')
+const path = require('path')
+
+const mkdirp = require('mkdirp')
+const rimraf = require('rimraf')
+const { facCaller } = require('./helper')
+
+const Fac = require('../')
+const _ = require('lodash')
+const crypto = require('crypto')
+
+const { Chance } = require('chance')
+
+const tmpDir = path.join(__dirname, 'tmp')
+
+let fac
+const chance = new Chance()
+beforeEach((done) => {
+  rimraf.sync(tmpDir)
+  mkdirp.sync(tmpDir)
+
+  Fac.ctx = { root: '' }
+
+  fac = new Fac(facCaller, {
+    db: path.join(__dirname, 'tmp', 'test.db'),
+    dirConf: path.join(__dirname, 'fixtures'),
+    runSqlAtStart: [
+      'CREATE TABLE IF NOT EXISTS Employees (id INTEGER PRIMARY KEY ASC, name, surname)'
+    ]
+  })
+
+  fac._start(done)
+})
+
+afterEach((done) => {
+  fac._stop(() => {
+    rimraf.sync(tmpDir)
+    done()
+  })
+})
```

Comment:
if you leave these outside of `describe('queries',` test are failing because these gets executed only once before all describe etc and returning this error: `Error: SQLITE_READONLY: attempt to write a readonly database`

Reply:
Thanks!